src/near2far.cpp

```diff
@@ -323,11 +324,14 @@ realnum *dft_near2far::get_farfields_array(const volume &where, int &rank, size_
       x.set_direction(dirs[1], where.in_direction_min(dirs[1]) + i1 * dx[1]);
       for (size_t i2 = 0; i2 < dims[2]; ++i2) {
         x.set_direction(dirs[2], where.in_direction_min(dirs[2]) + i2 * dx[2]);
-        if (!quiet && wall_time() > start + MEEP_MIN_OUTPUT_TIME) {
+        double t;
+        if (!quiet && (t = wall_time()) > start + MEEP_MIN_OUTPUT_TIME) {
```

Comment:
If we want this to work in the parallel case then we may need to add:
```
if (!quiet) all_wait();
```
to make the timing accurate on the master process.

The question is, what is the performance impact of putting synchronization in the loop like this?  I'm not sure